controllers/common/networkchaos/netem/types.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package netem
+
+import (
+	"context"
+
+	"github.com/pingcap/chaos-mesh/controllers/networkchaos/netem"
```

Comment:
group this import?

Reply:
I execute `make groupimports` locally and there is no change. Is this something wrong?

Reply:
It is wield 😯

Reply:
It seems to be default behavior https://github.com/golang/go/issues/27365#issuecomment-422720475 and https://github.com/golang/go/issues/26846#issuecomment-412629549 you need to put imports together so goimport will treat them as a group and sort within the group. Otherwise it assumes you put them into different group for some reason, e.g. third party, protobuf etc.